lib/packs/private.rb

```diff
@@ -370,10 +370,6 @@ def self.add_readme_todo(package)
       ).returns(ParsePackwerk::Package)
     end
     def self.create_pack_if_not_exists!(pack_name:, enforce_privacy:, enforce_dependencies:, team: nil)
-      if PERMITTED_PACK_LOCATIONS.none? { |permitted_location| pack_name.start_with?(permitted_location) }
```

Comment:
@naveg Could we change this to check the packs config and ensure that the desired pack path is permitted via the configuration?

Reply:
That makes sense. I'll give it a go and update this PR

Reply:
fixed